src/main/java/duke/tasks/Task.java

```diff
@@ -0,0 +1,39 @@
+package duke.tasks;
+
+public abstract class Task {
+    protected String description;
+    protected boolean isDone;
+    protected String type;
+
+    public Task(String description, String type) {
+        this.description = description;
+        this.isDone = false;
+        this.type = type;
+    }
+
+    public Task markDone() {
```

Comment:
Should this method be named differently because `markDone` does not really tell us that a task will be returned?